README.md

```diff
@@ -245,6 +245,24 @@ postedData.json
 
 * if `postedData.json` doesn't exist on the filesystem when `/match/against/file` is requested, stubby will match post contents against `{"fallback":"data"}` (from `post`) instead.
 
+#### json
+
+* not used if `post` or `file` are present.
+* will be parsed into a JavaScript object.
+* allows you to specify a JSON string that will be deeply compared with a JSON request
+
+Although not required, it is recommended to also specify a `application/json` header requirement.
+
+```yaml
+-  request:
+      url: ^/match/against/jsonString$
+      headers:
+         content-type: application/json
+      json: '{"key1":"value1","key2":"value2"}'
```

Comment:
Since YAML is a superset of JSON, perhaps the `json:` values could be actual objects instead of strings? This would keep us from having to `JSON.parse` on every request